### DISCUSSION TITLE
Pre existing world

### DISCUSSION THREAD

**Discussion by Porps43** [Pryaxis/TShock#2166] [Q&A]
Is there a way to use a already created (and player edited) world via the .wld file or do you need to have a new world created when making a server for Raspberry pi?

**Comment by Syshein** [comment]
https://github.com/Pryaxis/TShock/discussions/2161#discussioncomment-133394

**Comment by Kojirremer** [comment]
> Is there a way to use a already created (and player edited) world via the .wld file or do you need to have a new world created when making a server for Raspberry pi?

I'm not familiar with Raspberry Pi hosting, but desktop TShock can load specific world files via the command line using the -world parameter.
So, if you have a launcher and your world file is in your root TShock folder, you could add -world YourWorldName.wld next to the executable. If the wld file is not in your TShock folder, then you will have to add the whole route instead of just the file name.
